### ISSUE TITLE
tarp 'move' mechanics don't match description

### ISSUE THREAD

**Issue by pahp** [UMDLARS/one_night_in_sf#214]
If you `move tarp`, this happens:

> The Unused Office
> This room is a small office, there is a wooden desk devoid of any identifying objects. One of the ceiling panels seems to be missing in here. Along the east
> wall there is a tarp laying on the floor, partially laying against the wall. There are 2 cans of paint here. The east wall seems to be still unpainted.
> 
> You can see a Tarp here.
> 
> > move tarp
> As you fold up and take the tarp you notice underneath it there was a flathead screwdriver!
> > i
> You're carrying a USB Drive, a flashlight, a Phillips Head Screwdriver, a Print out, a loose brass key, a IBM 5100, a Security Key Fob and a Task List.
> 
> > l
> The Unused Office
> This room is a small office, there is a wooden desk devoid of any identifying objects. One of the ceiling panels seems to be missing in here. Along the east
> wall there is a tarp laying on the floor, partially laying against the wall. There are 2 cans of paint here. The east wall seems to be still unpainted.
> 
> You can see a Tarp and a Flathead here.
> 
> >

The description says that I took the tarp, but the tarp is not in my inventory; it is still in the room. IMO, the description should be changed so that it doesn't say that the player takes the tarp (since the player doesn't want the tarp anyway).

**Comment by pahp** [comment]
Here's some current behavior in @libraun 's PR:
```
The Unused Office
This room is a small office; set in the middle is a wooden desk, devoid of any identifying objects. One of the ceiling panels seems to be missing in here. Along the east wall there is a tarp laying on the floor, partially laying against the wall next to two cans of paint. The east wall seems to be still unpainted.

You can see a Tarp here.

> take tarp
As you fold up and take the tarp you notice underneath it there was a flathead screwdriver!
> l
The Unused Office
This room is a small office; set in the middle is a wooden desk, devoid of any identifying objects. One of the ceiling panels seems to be missing in here. Along the east wall there is a tarp laying on the floor, partially laying against the wall next to two cans of paint. The east wall seems to be still unpainted.

You can see a Tarp and a Flathead here.

> i
You're carrying a Task List.

> 
```

It says I take the tarp, but it is not in my inventory.  I confirmed that I can take the flathead and it disappears, and if I try to take the tarp again, it won't let me. 

Maybe change the first message to something like:

```
As you begin to fold up the tarp you notice a flathead screwdriver on the floor! You stop taking the tarp.
```

**Comment by libraun** [comment]
Good catch, thanks for the heads up!

**Comment by pahp** [comment]
Of course! I just realized a problem with my suggestion though -- when the screwdriver is gone, the tarp will still be there. If you take it again, the message mentioning the screwdriver will be wrong. Would this all be easier if we just let the player take the damn tarp?

**Comment by libraun** [comment]
Are you still referring to my fix? My solution was to print something like,
“You don’t see any point to carrying this around” if the player doesn’t
have the screwdriver, else reveal the screwdriver. Now that I think about
it, that will need a lot of test cases, but am I missing something else?

**Comment by libraun** [comment]
I’ll have to go through and check it out again, but I don’t know that
letting the player keep the tarp is a good idea personally. If I were
allowed to keep the tarp, I would try everything imaginable to use that
tarp in some way.

(Unless we could make the tarp useful for something else?)

**Comment by libraun** [comment]
Aha, I think I see what you mean. Is it that “take tarp” after “take tarp”
should yield a different dialogue option?
